app/utils/register-form-fields.js

```diff
@@ -104,5 +105,9 @@ export const registerCustomFormFields = () => {
       displayType: 'http://lblod.data.gift/display-types/birthDateInput',
       edit: RDFGeboorteInput,
     },
+    {
+      displayType: 'http://lblod.data.gift/display-types/genderSelector',
```

Comment:
Now `'http://lblod.data.gift/display-types/conceptSchemeMultiSelectorWithoutMeta',` is not used anymore in our forms, should we remove them from this register method?